src/codeu/chat/client/core/Context.java

```diff
@@ -40,6 +44,12 @@ public UserContext create(String name) {
         new UserContext(user, view, controller);
   }
 
+  // Added according to CodeU tutorial by Priyanka Agarwal
+  // Created View1 Variable
```

Comment:
I would have a comment explaining what this function returns, not that it was added from a tutorial. The tutorial part is less important to somebody reading the code than a comment explaining what the function returns.